### DISCUSSION TITLE
brew service installed with root as user

### DISCUSSION THREAD

**Discussion by gihdz** [Homebrew/discussions#2617] [Casks] [answered]
Hi, I'm installing [postgresql@12](https://formulae.brew.sh/formula/postgresql@12) using `brew install postgresql@12`. When I try to run this service, for some reason is run with user `root` which is discouraged, causing the postgresql server to not start. I don't know why this specific package is being installed with user root as I have other packages too and all of them have my username. Any suggestion on what I should do is greatly appreciated!

I'm using MBP m1 with monterey.

**Comment by gromgit** [comment, marked as answer]
Sounds like either you or someone else with access to your machine tried to _start_ the DB server with `sudo brew services start postgresql@12`.

**Reply by gihdz** [reply]
Thanks for answering! 
I pretty much blame myself for that. So I guess I should just chown the .plist? I have tried reinstalling the package with no luck.

**Reply by gromgit** [reply]
I don't have an installation to test on, but try:
```
# Clean up the root-based setup
sudo brew services stop postgresql@12
sudo brew services cleanup

# And start it again without root
brew services start postgresql@12
```

**Reply by gihdz** [reply]
Worked like a charm! Thank you!

**Comment by NewAlexandria** [comment]
in my case, there was an issue with running the `brew`-installed postgres (14) after installing another postgres (15) via `asdf`.  I still haven't found a solution to this

**Reply by gromgit** [reply]
What do the `brew`-installed PostgreSQL logs say?
